tests/integration/src/test/java/org/mule/test/construct/FlowConfigurationFunctionalTestCase.java

```diff
@@ -315,10 +315,6 @@ public void testSplitAggregateMapFlow() throws Exception
         assertTrue(apple.isBitten());
         assertTrue(banana.isBitten());
         assertTrue(orange.isBitten());
-
-        assertNotNull(results[0].getProperty("key", PropertyScope.INVOCATION));
```

Comment:
I reviewed this, this behavior actually makes sense, split Map messages have a key invocation property in Mule 3, and while this is not support in Mule 4 currently (MULE-9502) there is no reason this should be returned as a response once the responses are aggregated.